src/main/resources/com/google/api/codegen/metadatagen/java/grpc/grpc_package.snip

```diff
@@ -25,20 +25,29 @@
   dependencies {
     compile "com.google.protobuf:protobuf-java:{@metadata.protoVersionBound.lower}"
     compile "com.google.api:api-common:{@metadata.apiCommonVersionBound.lower}"
+    @switch metadata.packageType
+    @case "GRPC_CLIENT"
+      @join dependency : metadata.protoPackageDependencies
+        compile "com.google.api.grpc:{@dependency.name}:{@dependency.versionBound.lower}"
+      @end
+    @end
```

Comment:
I prefer the more general approach here, where GRPC_COMMON could conceivably have its own protoPackageDependencies. Since it has an empty list for this, it doesn't need to be special-cased.